R/ggMarginal.R

```diff
@@ -155,10 +155,11 @@ ggMarginal <- function(p, data, x, y, type = c("density", "histogram", "boxplot"
   pb <- ggplot2::ggplot_build(p)
   
   # Pull out the plot title if one exists and save it as a grob for later use.
+  # Note: You can't have a subtitle without a title in ggplot2
   hasTitle <- (!is.null(pb$plot$labels$title))
   if (hasTitle) {
-    title <- getTitleGrob(p = p)
-    p$labels$title <- NULL
+    titleGrobs <- getTitleGrobs(p = p)
+    p$labels$title <- p$labels$subtitle <- NULL
```

Comment:
For style/readability, can you please break this into two separate assignment statements?